mixers/AutoPanMix/AutoPanMix.sc

```diff
@@ -120,8 +114,16 @@ AutoPanMix : BaseMix {
 				~makeInputBusses.value(server, maxClients, inputChannelsPerClient, outputChannelsPerClient);
 
 				// send synthDefs
-				this.sendSynthDefs(["JackTripPannedIn", "JackTripPersonalMixOut", "JackTripSimpleIn"]);
-				if (maxClients > 100, { this.sendSynthDef("JackTripSimpleMix"); });
+				if (autopan, {
+					this.sendSynthDef("JackTripPannedIn");
+				}, {
+					this.sendSynthDef("JackTripSimpleIn");
+				});
+				if (maxClients > 100, {
+					this.sendSynthDef("JackTripSimpleMixFromBus", [~firstPrivateBus]);
```

Comment:
Are the second parameters (`[~firstPrivateBus]`) are being passed to `this.sendSynthDef`?

Reply:
nope, i forgot to remove that after my first attempts failed.